features/admin.api-resources.v2/hooks/use-api-resources-page-content.tsx

```diff
@@ -0,0 +1,185 @@
+    isApiServer: boolean;
+    isMcpServer: boolean;
+    resourceEditBackButtonLink: string;
+    resourceEditBackButtonText: string;
+    resourceEditPageTitle: string;
+    resourceEditPath: string;
+    resourceSearchBarPlaceholder: string;
+    resourceServerEmptyListSubtitle: string[];
+    resourceServerListDescription: ReactNode;
+    resourceServerListPageTitle: string;
+    resourceServerListTitle: string;
+    scopesTabContent: ScopesTabContent;
+};
+
+type ResourceServerConfigMap = {
+    [key in ResourceServerType]: ResourceServerConfig;
+};
+
+const useApiResourcesPageContent = () => {
+
+    const { t } = useTranslation();
+    const { getLink } = useDocumentation();
+
+    const productName: string = useSelector((state: AppState) => state.config.ui.productName);
+
+    const resourceServerType: ResourceServerType = window.location.pathname.includes("mcp-servers")
+        ? ResourceServerType.MCP
+        : ResourceServerType.API;
+
+    function getResourceServerTypeDisplayName(resourceServerType: ResourceServerType) {
+        switch(resourceServerType) {
+            case ResourceServerType.API:
+                return t("extensions:develop.apiResource.resourceTypes.api");
+            case ResourceServerType.MCP:
+                return t("extensions:develop.apiResource.resourceTypes.mcp");
+        }
+    }
+
+    const content: Partial<ResourceServerConfigMap> = {
+        [ResourceServerType.API]: {
+            addNewResourceButtonText: t("extensions:develop.apiResource.addApiResourceButton"),
+            createResourceWizard: {
+                description: t("extensions:develop.apiResource.wizard.addApiResource.subtitle"),
+                displayNameHint: t("extensions:develop.apiResource.wizard.addApiResource.steps.basic.form.fields." +
+                                "name.hint", { productName }),
+                displayNamePlaceholder: t("extensions:develop.apiResource.wizard.addApiResource.steps.basic.form." +
+                    "fields.name.placeholder"),
+
+                hiddenSteps: [],
+                identifierPlaceholder: t("extensions:develop.apiResource.wizard.addApiResource.steps.basic." +
+                        "form.fields.identifier.placeholder"),
+                title: t("extensions:develop.apiResource.wizard.addApiResource.title")
+            },
+            defaultSearchFilter: `type eq ${ APIResourcesConstants.BUSINESS }`,
+            deleteResourceWizardContent: {
+                buttonText: t("extensions:develop.apiResource.tabs.general.dangerZoneGroup" +
+                    ".deleteApiResource.button"),
+                heading: t("extensions:develop.apiResource.tabs.general.dangerZoneGroup" +
+                    ".deleteApiResource.header"),
+                subHeading: t("extensions:develop.apiResource.tabs.general.dangerZoneGroup" +
+                    ".deleteApiResource.subHeading")
+            },
+            isApiServer: true,
+            isMcpServer: false,
```

Comment:
Can't we have one generic flag for these two ?

Reply:
Yes, we can use resourceServerType check the resource type. Addressed in [38800b3](https://github.com/wso2/identity-apps/pull/8146/commits/38800b3b5b1fadcfd8a49b23dc24c976964d03ad)